util/lock-contention-graph.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash -eu
+# Copyright 2025 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You can obtain a copy in the file LICENSE in the source distribution
+# or at https://www.openssl.org/source/license.html
+#
+# Script to graph logs produced by REPORT_RWLOCK_CONTENTION vs time
+# Usage: ./lock-contention-graph.sh <logfile>
```

Comment:
please add labels here too

Reply:
are you referring to the labels on the graph?  i.e. the x and y axes?  I can do that.

Reply:
On line 25 you do `LABELS=$2`; sorry for the bad definition